app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -191,6 +191,10 @@ def parse_updates(vim, parser, updated_objects)
       props          = uncached_props.present? ? cached_props.deep_merge(uncached_props) : cached_props
 
       parser.parse(managed_object, update_kind, props)
+    rescue => err
+      _log.warn("Failed to parse #{managed_object.class.wsdl_name}:#{managed_object._ref}: #{err}")
```

Comment:
This should make it easier to identify which object is hitting an error in the future